analytics/ga.go

```diff
@@ -45,17 +44,16 @@ type UserSession struct {
 // NewUserSession provides a setup UserSession struct with sane defaults
 func NewUserSession() *UserSession {
 	v := version.Version
-	ctx := context.Background()
-	c, _ := dockeradapter.New()
-	dv, _ := c.ServerVersion(ctx)
+	host, _, _ := dockeradapter.DockerEnvvars()
+	serverAPIVersion, uniqueID, _ := dockeradapter.ServerInfo(host)
```

Comment:
Any reason not to handle `err` returned here?

Reply:
We'll still submit the analytics without this data here rather than black hole it.